File: 1020.判断正负数/main.cpp
#include <iostream>

using namespace std;

int main()
{
    int a;
    cin>>a;
    if(a==0)
        cout<< "zero" <<endl;
    else if(a<0)
        cout<< "negative" <<endl;
    else
        cout<< "positive" <<endl;
    return 0;
}

File: 1020.判断正负数/Main.java
package com.company;


import java.util.Scanner;

import static java.lang.Math.abs;

public class Main {

    public static void main(String[] args) {
        // write your code here
        Scanner sc = new Scanner(System.in);
        String [] str = {"negative","zero","positive"};
        int K = sc.nextInt();
        System.out.print(str[K==0?1:K/abs(K)+1]);
    }
}

